dockerfiles/theia/src/main.js

```diff
@@ -31,6 +31,13 @@ else {
         theiaConfig = defaultConfig;
         let dep = theiaConfig.dependencies;
         for (let d of pluginList) {
+            if (d.indexOf(":") > -1) {
```

Comment:
Could you add some comments on what is performed so it would be easy to maintain this code ?

something like
```
// check if plugin has a version using format pluginName:versionNumber
```